btcec/schnorr/musig2/nonces.go

```diff
@@ -6,11 +6,13 @@ import (
 	"bytes"
 	"crypto/rand"
 	"encoding/binary"
+	"fmt"
```

Comment:
Commit message body could use a bit more detail.

Reply:
Oh I must have forgot, this is actually the "breaking" 0.4.0 change https://github.com/jonasnick/bips/commit/20ba03106d245d42375e65f739c58dd9074daf6e